roles/github/defaults/main.yml

```diff
@@ -9,7 +9,9 @@ github_registry_username: ${{ github.actor }}
 
 github_registry_password: ${{ secrets.GITHUB_TOKEN }}
 
-github_image_name: ${{ github.repository }}:latest
+github_image_name: kayobe
+
+github_image_tag: latest
```

Comment:
How are you expecting this to be used? Would we need to update the tag and regenerate all the workflows whenever we want to use a new image?

We also need to think about how this will work with upgrades where we may need to have multiple versions of the image in use at once.

Reply:
We would need to regenerate the workflows if we want to use something other than the latest image.
We could potentially move the tag into a GitHub variable so it would be set outside of the workflows and updated as needed.

Reply:
Or tag it based on the OpenStack release name?

Reply:
How would we manage this between `.automation` submodules where the Dockerfile resides?
Would we build for yoga and then swap to [zed](https://github.com/stackhpc/kayobe-automation/commit/151413b1ff9e945922cbdcc540c070d3c3b992a7) submodule and build again.

We could do something such as 

```yaml
container:
   image: registry.jack.compute.sms-lab.cloud:4000/kayobe:${{ vars.image_tag || latest }}
```

Where `vars.image_tag` could be set to yoga then swapped to zed and then deleted.